docs/configuration/module-config/external-notification.mdx

```diff
@@ -10,6 +10,8 @@ import TabItem from "@theme/TabItem";
 
 The External Notification Module will allow you to connect a buzzer, speaker, LED, or other device to notify you when a message has been received from the mesh network. You can enable up to 3 pins independently from each other.
 
+The External Notification Module config options are: Enabled, Active, Alert Bell (General),Alert Bell Vibra, Alert Bell Buzzer, Alert Message (General), Alert Message Vibra, Alert Message Buzzer, Output (General), Output Vibra, Output Buzzer, Output Milliseconds, Use PWM, and Nag Timeout. External Notification config uses an admin message sending a `ConfigModule.ExternalNotificationConfig` protobuf.
```

Comment:
I'd like to see this first part written as a list, it's a little hard to read in this format.

Reply:
@charminULTRA  Currently, most of the config pages have their options listed in paragraph form.  I think the layouts could all use an overhaul but for now I was trying to keep everything uniform.

Reply:
Sounds good, I'll approve and merge.